lerobot/common/robot_devices/robots/manipulator.py

```diff
@@ -26,7 +27,7 @@
 # In nominal degree range ]-180, +180[
 ZERO_POSITION_DEGREE = 0
 ROTATED_POSITION_DEGREE = 90
-GRIPPER_OPEN_DEGREE = 35.156
+KOCH_GRIPPER_OPEN_DEGREE = 35.156
```

Comment:
FYI: Maybe not for this PR but I have found that I want to configure this on several occasions. Eg: for PushT, I want it to be 0. Or if picking a larger object, I might want it larger.

Reply:
Resolving, as this feature has now been developed and merged.